README.md

```diff
@@ -1,8 +1,10 @@
-# ![Alt text](docs/tersets.jpg "TerseTS")
+<center><img src="docs/tersets.jpg" align="middle" width="200px"/></center>
```

Comment:
It is unclear what the effect of this PR is and why "# TerseTS" has been removed. The [commit version of the README.md](https://github.com/cmcuza/TerseTS/blob/25e1559dbd516c18e96e7ee67d3aa838bc3cff55/README.md) and the [main version of the README.md](https://github.com/cmcuza/TerseTS/blob/main/README.md) seems to be identical.

Reply:
The main difference is setting the logo in the center and removing "#TerseTS" to reduce duplicates.

Reply:
As stated above, I cannot see any difference when looking at README.md for the [main branch](https://github.com/cmcuza/TerseTS/tree/main) compared to the [doc/update_readme branch](https://github.com/cmcuza/TerseTS/tree/doc/update_readme). I could not find many repositories that use logos for the title, but [here](https://github.com/neovim/neovim) [are](https://github.com/vim/vim) [a](https://github.com/ziglang/zig) few examples that may help to position and/or resize the logo. Also the logo should preferable be vector graphics in the form of a SVG file as it can be exported to a JPEG or PNG but the reverse is not possible.